composer.json

```diff
@@ -16,7 +16,7 @@
         }
     ],
     "require": {
-        "php": "^7.4|^8.1"
+        "php": "^7.4|^8.0"
```

Comment:
Changing the PHP version requirement can solve the following message when running the `composer update` command with the `PHP 8.0` version:

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires php ^7.4|^8.1 but your php version (8.0.26) does not satisfy that requirement.

Error: Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires php ^7.4|^8.1 but your php version (8.0.26) does not satisfy that requirement.

Error: Process completed with exit code 2.
```